tests/test.ts

```diff
@@ -87,31 +87,72 @@ describe("Basic minting", function () {
 
   it("Should not be allowed minting", async function () {
     const ipfsHash = '0xabcd1234'
-    await expect(remix.connect(betatester).safeMint(betatester.address, 'Beta Tester', '0.22.0', ipfsHash, 2)).to.be.revertedWith('is missing role 0x0000000000000000000000000000000000000000000000000000000000000000')
+    await expect(remix.connect(betatester).safeMint(betatester.address, 'Beta Tester', '0.22.0', ipfsHash, 2)).to.be.revertedWith('Caller is neither admin nor a challenge provider')
   });
 });
 
 
 describe("Challenge", function () {
+  let remixChallenges
+
+  it("Deploy Challenge with proxy", async function () {
+
+    const Remix = await ethers.getContractFactory("RemixChallenges");    
+    remixChallenges = await Remix.connect(owner).deploy();
+    await remixChallenges.deployed()
+
+    const implAddress = remixChallenges.address
+    console.log('implementation address', implAddress)
+
+    const Proxy = await ethers.getContractFactory('ERC1967Proxy')
+    const proxyChallenges = await Proxy.connect(owner).deploy(implAddress, '0x8129fc1c')
+    await proxyChallenges.deployed()
+    console.log("Remix reward deployed to:", proxyChallenges.address);
+
+    remixChallenges = await ethers.getContractAt("RemixChallenges", proxyChallenges.address)
+    remixChallenges = remixChallenges.connect(owner)
+
```

Comment:
May be you should run some assertion (`assert, expect etc`) to ensure that tx is completed in desired way